src/components/DataDecorator/Decorator/Decorator.js

```diff
@@ -153,9 +159,8 @@ Decorator.defaultProps = {
   active: false,
   className: '',
   href: undefined,
-  inert: false,
   inline: false,
-  onClick: () => {},
+  onClick: undefined,
```

Comment:
Need to be `undefined` so that the `onClick` prop can be properly assessed as "not provided" when determining what type of element to render the Decorator as.